launch/launch/actions/execute_process.py

```diff
@@ -180,15 +189,12 @@ def __init__(
         self.__prefix = normalize_to_list_of_substitutions(
             LaunchConfiguration('launch-prefix', default='') if prefix is None else prefix
         )
-        self.__output = output if output is not None else 'log'
-        allowed_output_options = ['log', 'screen']
-        if self.__output not in allowed_output_options:
-            raise ValueError(
-                "output argument to ExecuteProcess is '{}', expected one of [{}]".format(
-                    self.__output,
-                    allowed_output_options,
-                )
-            )
+        self.__output = output
+        # TODO(hidmic): consider implementing this using logging.LoggerAdapter instead
```

Comment:
What's the reason to not do it now? Time, complexity, or not sure if it's the right thing to do? I don't mind deferring it, I'm just curious.

Reply:
My preference would be a non-relative import, as the alternative names I'd suggest are just as long, i.e. `launch.logging.*` over `launch_logging.*`.

Reply:
> What's the reason to not do it now?
I was unsure if that's what you wanted. Right now we're using a transfer callback w/ no launch context. Using the logger adapter would instead provide a substitution framework, with it's advantages and restrictions.

> My preference would be a non-relative import

Non-relative import it is then.

Reply:
Weird, I posted the comment in the wrong place 🙃, sorry for the confusion.

Reply:
Re: the TODO

I don't mind, but I would like to remove or rephrase the TODO so it has a clear "do X after condition Y". Otherwise it will be here forever most likely.

Reply:
Alright, I'll take a shot at substitutions.

Reply:
Alright, not even a `LoggerAdapter`. This is much much simpler and will probably do for a long while. I thought at some point of enabling arbitrary output preprocessing and filtering but it was overkill.